src/Core/Domain/Address/Command/AddCustomerAddressCommand.php

```diff
@@ -271,9 +273,9 @@ public function getAddress2(): ?string
     }
 
     /**
-     * @return StateId|null
+     * @return StateIdInterface|null
```

Comment:
From the constructor it seems `$this->stateId` is never null The input parameter is optional but the class field is always created regarless of the input value So I think you can drop the `|null` and `?` before return type